cmd/config-sync/config_sync.go

```diff
@@ -171,3 +202,257 @@ func (c *ConfigSync) syncConfig(desired *qdr.BridgeConfig) error {
 	}
 	return nil
 }
+
+func (c *ConfigSync) syncRouterConfig(desired *qdr.RouterConfig) error {
+	agent, err := c.agentPool.Get()
+	if err != nil {
+		return fmt.Errorf("Could not get management agent : %s", err)
+	}
+	var synced bool
+	for i := 0; i < 3 && err == nil && !synced; i++ {
```

Comment:
Why not delegating the retry mechanism to the rate limited queue `*ConfigSync.events` ?

Reply:
Done, thanks for mentioning this.